araproc/analysis/standard_reco.py

```diff
@@ -791,20 +780,15 @@ def min_frac_corr_depth(self, corr_map, max_corr, fraction=0.6):
 
         min_depth = -float('inf')  # Initialize to find the shallowest depth (least negative)
```

Comment:
Can we move this to an `else` below now? I think it'll just be a little clearer that way now.

Reply:
I think you did this.